frontend/src/Util/DisplayWarning.jsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Snackbar,
+} from "@material-ui/core";
+import MuiAlert from '@material-ui/lab/Alert';
+
+export const DisplayInfo = (props) =>{
+  const {
+    displayMessage,
+    displaySuccess,
+    displayWarning,
+    successMessage,
+    failedMessage,
+    WarningMessage,
+    handleCloseMessage,
+  } = props
```

Comment:
Does this just add new items to props?

Reply:
This is "destructuring" the props. If we don't use destructuring, how we access data inside props would be `props.failedMessage`, `props.WarningMessage`, `props.handleCloseMessage` and so on. With uses of destructuring, we can access them by simply calling `failedMessage`, `WarningMessage`, `handleCloseMessage` and so on. Another benefit of being this explicit is that this helps readers know what are in the props.

Reply:
(Additional comment: this does not add new items to props. it only demonstrates what have been passed from caller, i.e. parent component, into this DisplayInfo component.

Reply:
Interesting.